relay-general/src/processor/chunks.rs

```diff
@@ -157,19 +157,28 @@ where
 
 /// Splits the string into chunks, maps each chunk and then joins chunks again, emitting
 /// remarks along the process.
-pub fn process_chunked_value<F>(value: &mut String, meta: &mut Meta, f: F)
+pub fn process_chunked_value<F>(value: &mut String, meta: &mut Option<&mut Meta>, f: F)
```

Comment:
I would particularly like some input on this function

Reply:
For simplicity's sake, I would leave this function as it was, and call it with an empty `Meta::default()` instead. We could even do this on the highest level, i.e. call `apply_all_rules` with a `Meta::default()` instead of `None`.